packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
-## NEXT
+## 2.2.0
 
 * Fixes violations of new analysis option use_named_constants.
 * Fixes avoid_redundant_argument_values lint warnings and minor typos.
+* Updates `useAndroidViewSurface` to require Hybrid Composition, making the
```

Comment:
nit: Should this include that the minimum Flutter version was raised to 3.0.0.

Reply:
Can't hurt certainly! Done.